ui/components/MesheryFilters/FiltersGrid.js

```diff
@@ -141,7 +141,7 @@ function FiltersGrid({ filters=[],handleDeploy, handleUndeploy, handleClone, han
         isDelete={!modalOpen.deploy}
         title={ modalOpen.name }
         componentCount = {modalOpen.count}
-        tab={modalOpen.deploy ? 0 : 1}
+        tab={modalOpen.deploy ? 2 : 1}
```

Comment:
I'm ignorant here. What does "2" signify here?

Reply:
2 is the tab number for deploy in the confirmation modal. 
![Screenshot 2023-02-16 104303](https://user-images.githubusercontent.com/103871275/219274742-fba34d22-ffeb-4005-8cf7-c9cf363db3cb.png)
Actually, it is a change made in my previous PR.